modules/ROOT/pages/administration/access-control/dbms-administration.adoc

```diff
@@ -71,13 +71,13 @@ This section explains how to use Cypher to manage Neo4j DBMS administrative priv
 
 All DBMS privileges are relevant system-wide.
 Like user management, they do not belong to one specific database or graph.
-For more details on the differences between graphs, databases and the DBMS, refer to xref::introduction/neo4j-databases-graphs.adoc[].
+For more details on the differences between graphs, databases and the DBMS, refer to xref::introduction/cypher_neo4j.adoc[].
```

Comment:
why switching from `neo4j-databases-graphs`? that is still a document (you have edited it) and it contains the differences

Reply:
Some of the old pages for the introduction are currently still in the repo - they will be deleted as soon as I am done with the necessary restructuring stages.